### PR TITLE
Docs readme revamp

### PR DESCRIPTION
**Pull request by alexcg1**
Categorization, adding background reading, adding more examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,124 +59,269 @@
...
+<td><h3>🚀</h3></td>
+</tr>
+
+<tr>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/mnist-with-mindspore">Search MNIST with Mindspore</a></h4>
+Search handwriting examples using Mindspore
+</td>
+<td><h3>🚀</h3></td>
+</tr>
```
Comment (nan-wang) [inline README.md:144]: Let's move mindspore to `Extract Features and Vectors` and rename that section to `Misc`. I'm not sure about the name.

File: README.md [hunk 2]
```diff
@@ -47,124 +59,269 @@
...
+  <td>
+    <h4>
+      <a href="https://github.com/jina-ai/examples/tree/master/object-search">
+        Object detection with fasterrcnn and MobileNetV2
+      </a>
+    </h4>
+    Detect, index and query similar objects
+  </td>
+  <td><h3>🚀</h3></td>
+</tr>
```
Comment (nan-wang) [inline README.md:240]: I'd suggest to move this to `Search Images`

File: README.md [hunk 3]
```diff
@@ -47,124 +59,269 @@
...
+<h4><a href="https://github.com/jina-ai/examples/tree/master/cross-modal-search">Search images from captions and vice-versa</a></h4>
+Use one modality (text) to search another (images)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 </td>
 <td><h3>🐣</h3></td>
 </tr>
 
+</table>
+
+### Extract Features and Vectors
```
Comment (nan-wang) [inline README.md:213]: Rename this section to `Misc`?

File: README.md [hunk 4]
```diff
@@ -47,124 +59,269 @@
...
+<tr>
+<th width="90%" style="width: 90%">Tutorials</th>
+<th width="10%" style="width: 10%">Level</th>
+</tr>
+
 <tr>
 <td>
-<h4><a href="https://github.com/jina-ai/examples/tree/master/flower-search">Build a Flower Image Search System</a></h4>
-Learn how to build an image search system and define you own executors and run them in docker
+<h4><a href="https://github.com/jina-ai/examples/tree/master/fashion-example-query">Search by Category Using Query Language</a></h4>
```
Comment (nan-wang) [inline README.md:223]:
```suggestion
<h4><a href="https://github.com/jina-ai/examples/tree/master/fashion-example-query">Use Query Language</a></h4>
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
Minor comments

File: README.md (hunk 4 above)
Comment (alexcg1) [reply README.md:223]: I think we need to explain WHY use Query Lang or what it can DO

Comment (alexcg1) [reply README.md]: Done!

**Comment by alexcg1** [comment]
@nan-wang good to merge now?

File: README.md [hunk 5]
```diff
@@ -47,90 +59,220 @@
...
+<h4><a href="https://github.com/jina-ai/examples/tree/master/tumblr-gif-search">Search Tumblr GIFs with KerasEncoder</a></h4>
+Use prefetching and sharding to improve the performance of your index and query flow when searching animated GIFs.
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 </td>
 <td><h3>🐣</h3></td>
 </tr>
 
+</table>
+
+### Cross and Multi Modal Search
```
Comment (JoanFM) [inline README.md:197]: I would put `Cross Modality Search`. A new case for `MultiModal` is coming. Like this we do make a clear distinction

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (alexcg1) [reply README.md]: Done

**Review by JoanFM — APPROVED** [review]

**Comment by alexcg1** [comment]
Ping @nan-wang

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**